src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    final String feedbackToUser;
```

Comment:
this is not complete encapsulation, why?
what is the access level for this variable now?